bin/utils/generate-local-user-mappings.sh

```diff
@@ -60,5 +62,28 @@ function create_user_by_email_responses() {
   rm $users_ids_tmp_file
 }
 
+function create_orgs_by_email_responses() {
+  rm -rf $mock_org_by_email_dir
+  mkdir $mock_org_by_email_dir
```

Comment:
would it be better to just do
```suggestion
  rm -rf ${mock_org_by_email_dir}/*
```
so you remove the contents of the directory without deleting the directory itself

Reply:
but then i need to ensure that dir exists,  create it or commit to repo which i would like to avoid

Reply:
makes sense, didn't think of that part